ios/NewExpensify.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
```

Comment:
Same with this file too @aldo-expensify

Reply:
Likewise for this file it was automatically created by XCode 14 - let me know if you want it removed - it doesn't appear to be required to run the project from XCode